src/legacy/core_plugins/data/public/search/aggs/agg_config.ts

```diff
@@ -435,9 +400,6 @@ export class AggConfig {
 
     // clear out the previous params except for a few special ones
     this.setParams({
-      // split row/columns is "outside" of the agg, so don't reset it
-      row: this.params.row,
```

Comment:
special parameter used for figuring out if we should do a row or column split (with data table vis). this is not relevant to data querying at all so this setting was moved to vis.params